### PR TITLE
Bug/3633/copyright translation

### PR DESCRIPTION
**Pull request by pcoder**
Requirement:
Compile the .po files.

To test this PR.
1. Check all the pages in DCL in DE, the footer should contain "Alle Rechte vorbehalten"
2. Check all the pages of hosting app in DE, the footer should contain "Alle Rechte vorbehalten"

### CODE REVIEW

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -587,6 +571,21 @@ msgid ""
...
+#~ msgstr "So funktioniert es"
+
+#~ msgid "Your infrastructure"
+#~ msgstr "deine Infrastruktur"
+
+#~ msgid "Our inftrastructure"
+#~ msgstr "Unsere Infrastruktur"
+
+#~ msgid "Pricing"
+#~ msgstr "Preise"
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: Preise->Preis

Comment (pcoder) [inline hosting/locale/de/LC_MESSAGES/django.po]: `584 #~ msgstr "Preise"` is a commented out line. Anyways, I will make this change for future reference.

File: datacenterlight/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -496,11 +500,23 @@ msgstr "Unser Angebot beginnt bei 15 CHF pro Monat. Probier's jetzt aus!"
 msgid "Actions speak louder than words. Let's do it, try our VM now."
 msgstr "Tagen sagen mehr als Worte – Teste jetzt unsere VM!"
 
-#: views.py:235
+#: views.py:199
```
Comment (pcoder) [inline datacenterlight/locale/de/LC_MESSAGES/django.po]: nico11:30 PM suggests
```
"Invalid number of cores" / "Invalid RAM size" / "Invalid storage size"
"Ungültige Anzahle CPU-Kerne" / "Ungültige RAM-Grösse" / "Ungültige Speicher-Grösse" 
```

**Comment by pcoder** [comment]
I made those validation text changes @verysanghee and Preise -> Preis @verysanghee

**Comment by verysanghee** [comment]
tested and looks good

**Pull request merged**